### PR TITLE
Paracompact spaces verifying the CCC are Lindelöf

### PR DESCRIPTION
**Pull request by Almanzoris**
Proposed theorem: T_2 Paracompact spaces verifying the CCC are Lindelöf.

Maybe $T_2$ could be weakened to $R_1$.

### CODE REVIEW

**Comment by Almanzoris** [comment]
Reading the questions... if I have understood well, the theorem would be: Collectionwise normal + CCC == > Lindelöf, right?

**Comment by prabau** [comment]
As explained in https://math.stackexchange.com/questions/4862414, the full set of implications is:

paracompact T2 ⇒ paracompact regular ⇒ paracompact normal = fully normal ⇒ paracompact.

This is already reflected in pi-base.  So a good weakening of the hypotheses would be [Fully normal + CCC] as you suggest.

Does [Paracompact + CCC ==> Lindelof] hold in general?

**Comment by Almanzoris** [comment]
From what I've read (https://math.stackexchange.com/q/1301345). The space is assumed to be $T_2$ in the proof of (https://math.stackexchange.com/q/190078).

**Comment by prabau** [comment]
I was just asking about paracompact in general, to keep it in the back of our minds.  But for now, we can go with what you have, with fully normal in the assumption.

**Comment by yhx-12243** [comment]
I think we don't need $T_2$, see https://dantopology.wordpress.com/2009/10/18/ccc-paracompact-lindelof/ .

However paracompact is definitely **not** to weaken as metacompact.

**Comment by StevenClontz** [comment]
I'd approve of adding para-Lindelof, particularly if we can improve (probably on Math SE) Dan's proof to show para-Lindelof+ccc=>Lindelof.

**Comment by prabau** [comment]
@yhx-12243 I tried to read Dan Ma's proof in his dantopology blog and I am missing one thing.

(*) Given $E\in\mathcal C$, why is it that $T(E)$ is countable?

I understand that each element of $\mathcal C$ meets only finitely many elements of $\mathcal{U}_0$.  But it seems it could be possible that an element $C\in\mathcal C$ meets uncountably many elements of $\mathcal C$.  So (*) is not clear.

File: theorems/T000630.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+then:
+  P000018: true
+refs:
+- mathse: 190078
+  name: Every paracompact space with the Suslin property is Lindelöf
+- mathse: 1301345
+  name: How to prove that every Paracompact space with the Suslin property is Lindelof
+---
+
+See {{mathse:190078}} and {{mathse:1301345}}.
```
Comment (StevenClontz) [inline theorems/T000630.md:17]:
```suggestion
---
uid: T000630
if:
  and:
  - P000003 : true
  - P000030: true
  - P000029: true
then:
  P000018: true
refs:
- mathse: 5004462
  name: Is every paracompact CCC space Lindelöf?
---

See {{mathse:5004462}}.
```

We can update this theorem in a separate PR to use para-Lindelof (when someone wants to add para-Lindelof).

Of course this suggestion assumes my proof at Math SE is valid, so wait until it's vetted better before taking this suggestion.

Comment (StevenClontz) [reply theorems/T000630.md:17]: I think @david20000813 found an error, which I think mirrors @prabau's concern with Dan Ma's blog post.

So this suggestion is certainly premature.

**Review by StevenClontz — CHANGES_REQUESTED** [review]
I think, barring a quick answer at https://math.stackexchange.com/questions/5004462, having `fully normal + ccc => lindelof` as this PR is the best way to go forward. Please add the note that the two posts we reference use "paracompact" to mean what pi-Base calls "fully normal".

EDIT: Now https://math.stackexchange.com/questions/5004462 justifies simply `paracompact + ccc => lindelof`.

**Comment by prabau** [comment]
Don't the other posts assume Hausdorff as part of paracompact, which is an even stronger notion than "fully normal"?

**Comment by yhx-12243** [comment]
Yes $\sigma$-discrete refinement only requires fully normal.

What about the proof in https://ncatlab.org/nlab/show/paracompact+spaces+satisfying+the+countable+chain+condition+are+Lindel%C3%B6f using AC?

**Comment by prabau** [comment]
@yhx-12243 A superficial reading of the nlab link would seem to indicate the result is true.  But one has to check the details.

Would be be able to add an answer to  https://math.stackexchange.com/questions/5004462, giving more details?  In particular, show in detail the validity of the Zorn lemma argument here, and the various sentences after that?

**Comment by StevenClontz** [comment]
Yeah this has been keeping me up particularly after my goof trying to adapt Ma's argument; I'm working on details in an answer if no one beats me too it. 🙃

**Comment by StevenClontz** [comment]
https://math.stackexchange.com/a/5004482/86887 I think the nlab came through for us tonight :-D

**Comment by david20000813** [comment]
Nice proof! I was convinced there would be a counterexample and have been trying to find one for the past two hours. Now I know why my efforts failed 😂

File: theorems/T000630.md (hunk 1 above)
Comment (StevenClontz) [reply theorems/T000630.md:17]: The Math SE post now has a valid proof, so this suggestion may now be considered.

**Comment by prabau** [comment]
@StevenClontz

By T129 we alreadyknow that CCC implies  weakly Lindelof. (That's https://math.stackexchange.com/questions/4743458).

So a stronger theorem than the proposed [CCC + paracompact ==> Lindelof]
would be [weakly Lindelof + paracompact ==> Lindelof].

The proof is even easier that what you have right now (assuming mathse:4743458).
Would it be possible to amend your mathse post accordingly, and then we can refer to it here?

**Comment by prabau** [comment]
Well, in the interest of moving things along, I'll try to put this information in another answer to the mathse post.

**Comment by prabau** [comment]
> Every paracompact weakly Lindelöf space is Lindelöf.

Answer now posted in https://math.stackexchange.com/5005198.

**Review by StevenClontz — APPROVED** [review]
Thanks again for the improvement @prabau!

**Pull request merged**